crypto_kem/ntruhrss701/clean/sample.c

```diff
@@ -37,14 +37,14 @@ void PQCLEAN_NTRUHRSS701_CLEAN_sample_iid_plus(poly *r, const unsigned char unif
 
     /* s = <x*r, r>.  (r[n-1] = 0) */
     for (i = 0; i < NTRU_N - 1; i++) {
-        s += r->coeffs[i + 1] * r->coeffs[i];
+        s += (uint16_t)((uint32_t)r->coeffs[i + 1] * (uint32_t)r->coeffs[i]);
```

Comment:
Did you want to use the `OVERFLOWING_MUL` macro here for consistency with the other implementations?

Reply:
In SABER, it occurred so often that I thought it's more clear if I add a macro. Here it's only in two places – I don't think a macro makes it much more clear.